inflammation/models.py

```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```

Comment:
Unclear function name. Use a better name.

Reply:
Agreed, think I'll change it to 'daily_standard_deviation' to keep convention with other functions in models.py.